### PR TITLE
POC - add route for verifying user location

### PR DESCRIPTION
**Pull request by kyle4908**
Pass the following in the request body:
{
    "userId": String,
    "longitude": number,
    "latitude": number
}

**User does not exist:**
Screenshot (873)

**User exists but is not in the correct place:**
Screenshot (874)

**User exists and is at the correct spot:**
Screenshot (875)

### CODE REVIEW

File: src/poc/server/poc-server/src/controllers/users.ts [hunk 1]
```diff
@@ -12,7 +17,50 @@ export const createSession = async (req: Request, res: Response) => {
...
+
+        if (distance > 10) {
+            console.log(
+                "User [%s], is too far from the site, failed to authenticate",
+                uuid
+            );
+            res.status(401).json({ message: "User is not on a registered site" });
+            return;
+        }
+        res.status(200).json({ token: "" });
```
Comment (kyle4908) [inline src/poc/server/poc-server/src/controllers/users.ts:59]: did we want to return a token from this? @Spitgranger @akshitg3

Comment (Spitgranger) [reply src/poc/server/poc-server/src/controllers/users.ts:59]: Ya I think that's how we're gonna tell if a user is authorized or not after they provide their location

File: src/poc/server/poc-server/src/controllers/users.ts [hunk 2]
```diff
@@ -12,7 +17,50 @@ export const createSession = async (req: Request, res: Response) => {
...
-    res.status(200).json({uuid});
+    res.status(200).json({ uuid });
+}
+
+/**
+ * Route to verify a users location
+ * @name /api/users/verify-location
+ * @function
+ */
+export const verifyLocation = async (req: Request, res: Response) => {
```
Comment (Spitgranger) [inline src/poc/server/poc-server/src/controllers/users.ts:28]: this endpoint should also emit an event to the socket which has the connection that is the same as the uuid provided and send a JWT either through the socket or in the response

File: src/poc/server/poc-server/src/controllers/users.ts [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+import * as jose from 'jose';
 import { io } from "../index";
 export const sessionIds: string[] = [];
+export const ITBLocation = {
+    latitude: 43.25898815315478,
+    longitude: -79.92086378832343
+};
+
+dotenv.config();
+const secretKey = createSecretKey(process.env.JWT_SECRET, 'utf-8');
```
Comment (kyle4908) [inline src/poc/server/poc-server/src/controllers/users.ts:15]: the JWT_SECRET will have to exist in a .env file, not sure if there's a better way to store the secret

File: src/poc/server/poc-server/src/controllers/users.ts (hunk 2 above)
Comment (kyle4908) [reply src/poc/server/poc-server/src/controllers/users.ts:34]: added

**Review by Spitgranger — APPROVED** [review]
lgtm

**Pull request merged**